src/checks/set_root_password.ts

```diff
@@ -1,48 +1,24 @@
 import { it, page } from "../lib/helpers";
-import { ElementHandle } from "puppeteer-core";
 
 // defines a test for configuring the root password from the main Agama screen
 export function setRootPassword(password: string) {
   it("allows setting the root password", async function () {
     await page.locator("a[href='#/users']").click();
+    await page.locator("a[href='#/users/root/edit']").click();
 
-    const button: ElementHandle = await Promise.any([
-      page.waitForSelector("button::-p-text(Set a password)"),
-      page.waitForSelector("button#actions-for-root-password"),
-    ]);
-
-    await button!.click();
-
-    const id: string = await button!.evaluate((x) => x.id);
-    // drop the handler to avoid memory leaks
-    button!.dispose();
-
-    // if the menu button was clicked we need to additionally press the "Change" menu item
-    if (id === "actions-for-root-password") {
-      await page.locator("button[role='menuitem']::-p-text('Change')").click();
-    }
+    await page.locator("::-p-text(Use password)").wait();
```

Comment:
NP: By mistake, the "Use a password" thing is a "switch", which can be pick in React Testing Library by its role. Wondering if you can get by role here too.

And I say "by mistake" because it should be a checkbox instead, something to be fixed in future releases.